frontend/src/metabase/querying/metrics/components/MetricEditor/MetricEditorBody/MetricEditorBody.tsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import { ResizableBox } from "react-resizable";
+
+import { Notebook } from "metabase/querying/notebook/components/Notebook";
+import { Box, Flex } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+
+import S from "./MetricEditorBody.module.css";
+import { MetricEditorSidebar } from "./MetricEditorSidebar";
+import { ResizableBoxHandle } from "./ResizableBoxHandle";
+
+const NOTEBOOK_HEIGHT = 500;
+
+type MetricEditorBodyProps = {
+  question: Question;
+  reportTimezone: string;
+  isDirty: boolean;
+  isResultDirty: boolean;
+  isRunnable: boolean;
+  onChange: (question: Question) => Promise<void>;
+  onRunQuery: () => Promise<void>;
+};
+
+export function MetricEditorBody({
+  question,
+  reportTimezone,
+  isDirty,
+  isResultDirty,
+  isRunnable,
+  onChange,
+  onRunQuery,
+}: MetricEditorBodyProps) {
+  const [isResizing, setIsResizing] = useState(false);
+
+  return (
+    <ResizableBox
+      className={S.root}
+      axis="y"
+      height={NOTEBOOK_HEIGHT}
+      handle={<ResizableBoxHandle />}
+      resizeHandles={["s"]}
+      onResizeStart={() => setIsResizing(true)}
+      onResizeStop={() => setIsResizing(false)}
+    >
+      <Flex w="100%" style={{ overflow: isResizing ? "hidden" : "scroll" }}>
```

Comment:
in `DatasetNotebook` we have the overflow same fix, but:
- it's encapsulated in a function
- it has a nice explanatory comment
- it uses `overflowY`

Can we make this consistent and make sure the explanatory comment can be found from both places?

Reply:
Actually there's a bug - we should use `"auto"` instead of `"scroll"` (for both metrics and models).
With `"scroll"` the disabled scrollbars are always shown (it's more prominent in metrics editor because it uses `overflow` instead of `overflowY`).

![image](https://github.com/user-attachments/assets/d0eede92-7e30-4e1e-93fe-1631be7ca0bb)